### PR TITLE
test githut actions

### PR DESCRIPTION
**Pull request by dairoquintero**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -58,11 +58,13 @@ dependencies {
     // https://mvnrepository.com/artifact/org.aspectj/aspectjweaver
     implementation group: 'org.aspectj', name: 'aspectjweaver', version: '1.9.4'
 
-    implementation group: 'org.codehaus.groovy', name: 'groovy', version: '3.0.7'
-
+   implementation group: 'org.codehaus.groovy', name: 'groovy', version: '3.0.7'
+    // https://mvnrepository.com/artifact/org.codehaus.mojo/aspectj-maven-plugin
+    //implementation group: 'org.codehaus.mojo', name: 'aspectj-maven-plugin', version: '1.11'
```
Comment (aperdomob) [inline build.gradle]: remove

File: settings.gradle [hunk 2]
```diff
@@ -1,2 +1,2 @@
 rootProject.name = 'project_api_java'
-
+//weaver options="-XnoInline -verbose -showWeaveInfo -Xlint:ignore -Xset:weaveJavaPackages=true -Xset:weaveJavaxPackages=true
```
Comment (aperdomob) [inline settings.gradle]: is it needed?

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: settings.gradle (hunk 2 above)
Comment (dairoquintero) [reply settings.gradle]: It's not needed, excuse me, i just trying to solve the warnings. I will delete this line

File: build.gradle (hunk 1 above)
Comment (dairoquintero) [reply build.gradle]: done

**Review by dairoquintero — COMMENTED** [review]
Done

File: build.gradle [hunk 3]
```diff
@@ -58,11 +58,12 @@ dependencies {
     // https://mvnrepository.com/artifact/org.aspectj/aspectjweaver
     implementation group: 'org.aspectj', name: 'aspectjweaver', version: '1.9.4'
 
-    implementation group: 'org.codehaus.groovy', name: 'groovy', version: '3.0.7'
+   implementation group: 'org.codehaus.groovy', name: 'groovy', version: '3.0.7'
```
Comment (aperdomob) [inline build.gradle]: restore the space in this line

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 3 above)
Comment (dairoquintero) [reply build.gradle]: Done

**Review by aperdomob — APPROVED** [review]

**Pull request merged**